### DISCUSSION TITLE
[11 Settembre 2023] Cache

### DISCUSSION THREAD

**Discussion by FeddyLix17** [sapienzastudentsnetwork/architettura-degli-elaboratori#64] [Cache (M-Z)]
![image](https://github.com/sapienzastudentsnetwork/archelab2223/assets/66442720/00020545-af0c-4717-bd91-fda33c517ef7)

**Comment by AglaiaNorza** [comment]
![image](https://github.com/sapienzastudentsnetwork/architettura-degli-elaboratori/assets/75022526/5b338078-fe85-4a19-886d-ba5a2b9d1137)

dovrebbe essere così
